SyntheticsRunTestsTask/__tests__/fixtures.ts

```diff
@@ -33,8 +34,11 @@ const runMockedTask = (mockName: string): MockTestRunner => {
     throw Error(`The mocked task file does not exist: mocks/${mockName}.js\n` + 'Did you forget to run `yarn build`?')
   }
 
+  // See `16.15.0` in `.node-version`
```

Comment:
I added this comment to be able to easily grep for `16.15.0` and get all the places where the node version is used.